pkg/detectors/grafana/grafana.go

```diff
@@ -23,13 +24,22 @@ var _ detectors.Detector = (*Scanner)(nil)
 var (
 	defaultClient = common.SaneHttpClient()
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	keyPat = regexp.MustCompile(`\b(glc_[A-Za-z0-9+\/]{50,150}\={0,2})`)
+	keyPat = regexp.MustCompile(`\b(glc_eyJ[A-Za-z0-9+\/=]{60,160})`)
```

Comment:
I generate around 10 tokens and each one of them was detected by this regex. Earlier few tokens suffix was being trim due to 150 character limit.